src/main/webapp/index.html

```diff
@@ -167,6 +167,7 @@ <h1><a href="/">Census Viz</a></h1>
       <br>
       <div id="data-table"></div>
     </div>
+    <div id="history"></div>
```

Comment:
This is irrelevant to this line, but I wanted to comment in index.html. Can you add getHistory to the onload? Right now you have to make a query before you can see your previous history (but it maintains your history from other tabs you have open, which is great!) I'd like to see my history even before I make any queries, if I'm logged in.

Note that this might require you to add a check to the very beginning of getHistory that returns without doing anything if the user is logged out. But I think that would be a smart thing to have anyway.

Reply:
^^ you might be able to do this by getting history in loginInit(). Also, if you're signed out then sign in on a query result history doesn't show until you make a new query, it might be good to also get history in the onSignIn function